capstone/src/main/resources/static/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Events</title>
+    <link rel="stylesheet" href="style1.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <h1>Events</h1>
+    <div id="filters">
+      <p>Filters</p>
```

Comment:
The format of the filters is a little confusing. Maybe place them in a column or put some spaces between them, but feel free to address this in a later PR since style changes is not in the scope of this PR.